projects/m2/014-sublists/js/index.js

```diff
@@ -0,0 +1,23 @@
+const prompt = require("prompt-sync")();
+
+
+function isSublist(){
+    const list = "25,30,35,40".split(",");
+    const sublist = "25,30,35".split(",");
+    const indexControl = [];
```

Comment:
Ensure you pass lists and sublists as parameters and not as internal values ​​to the function.
The Functions should be as abstract and reusable as possible.